Documentation.md

```diff
@@ -1027,6 +1027,68 @@ Here is a full conversation example :
 | createdAt | Creation date of the conversation | DateTime | ✓ | ISO-8601 |
 | updateAt | Date of the last message received | DateTime | ✓ | ISO-8601 |
 
+
+##### Get the proactive replies (endpoint)
```

Comment:
Est-ce qu'on peut rajouter que cette feature est utile à implem si isProactive = true ?

Reply:
Oui et non en fait. Le isProactive=true qui est important, c'est celui que l'on garde dans `BotUserState` de conversations-bot-service-impl. Il est inutile celui que l'on stock côté bot-builder-app ou bot-externe.

Il faudra peut-être qu'on fasse le ménage là dedans pour éviter la confusion par contre tu as raison.

Reply:
Outre l'aspect variable, je pense que ce serait pas mal d'avoir une petite phrase ou deux pour indiquer que cet endpoint n'est utile que dans le cadres de bots proactifs. Pour que les intégrateurs qui n'en ont pas l'usage la zap sans trop se poser de question.

Je ne connais pas en détail la structure de cette doc, ça pourrait être intéressant de présenter ces deux "modes" de bots en introduction de la section bot, pour introduire le pourquoi de cette route

Reply:
J'ai rajouté la phrase "*If you don't need your bot to be proactive, you can ignore the endpoint*"

Pour que les intégrateurs voient au premier coup d'oeil si ils ont besoin de ce endpoint ou pas.